.gitignore

```diff
@@ -6,3 +6,4 @@ dist
 .DS_Store
 *.log
 /package-lock.json
+.idea
```

Comment:
Please remove it from here and add to your global `.gitignore`

[global gitignore](http://egorsmirnov.me/2015/05/04/global-gitignore-file.html)